enhancements/cluster-logging/multi-cluster-observability.md

```diff
@@ -0,0 +1,425 @@
+---
+title: multi-cluster-observability-addon
+authors:
+  - "@periklis"
+  - "@JoaoBraveCoding"
+  - "@pavolloffay"
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Observability Addon
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-Cluster Observability has been an integrated concept in Red Hat Advanced Cluster Management (RHACM) since its inception but only incorporates one of the core signals, namely metrics, to manage fleets of OpenShift Container Platform (OCP) based clusters (See [RHACM Multi-Cluster-Observability-Operator (MCO)](rhacm-multi-cluster-observability)). The underlying architecture of RHACM observability consists of a set of observability components to collect a dedicated set of OCP metrics, visualizing them and alerting on fleet-relevant events. It is an optional but closed circuit system applied to RHACM managed fleets without any points of extensibility.
+
```

Comment:
Perhaps a little nit - but for sake of accuracy need to mention that current MCO is not `closed circuit system applied to RHACM managed fleets without any points of extensibility.`. In fact it uses the same addon framework that you propose to use below. And in fact the current MCO could be extended to incorporate both logging and tracing - at least technically.

Reply:
MCO is using addon framework? AFAIK it is an operator, or am I mislead by looking on this too narrow on this repo: https://github.com/stolostron/multicluster-observability-operator/

Reply:
In a typical managed cluster with ACM Observability enabled, you will see 9 separate addons 
![image](https://github.com/openshift/enhancements/assets/20711730/9d5b1385-fb0f-47dc-b68b-c9c1a1e94541)
MCO is a hub side component that polls and watches for new clusters being added
As a new managed cluster is found, the endpoint metrics collector addon (itself an operator) is rolled out